content/docs/application-mobility/deployment.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Deployment"
+linkTitle: "Deployment"
+weight: 1
+Description: >
+  Deployment for Dell Container Storage Modules (CSM) for Application Mobility
+---
+
+## Pre-requisites
+- [Request a License for Application Mobility](../../license/)
+- Object store bucket accessible by both the source and target clusters
+
+## Installation
+1. Create a namespace where Application Mobility will be installed.
+    ```
+    kubectl create ns application-mobility
+    ```
+1. Edit the license Secret file (see Pre-requisites above) and set the correct namespace (ex: `namespace: application-mobility`)
+1. Create the Secret containing a license file
+    ```
+    kubectl apply -f license.yml
+    ```
+1. Add the Dell Helm Charts repository
+    ```
+    helm repo add dell https://dell.github.io/helm-charts
+    ```
+1. Either create a values.yml file or provide the `--set` options to the `helm install` to override default values from the [Configuration](#configuration) section.
+1. Install the helm chart
+    ```
+    helm install application-mobility -n application-mobility dell/csm-application-mobility
+    ```
+
+
+### Configuration
+
+This table lists the configurable parameters of the Application Mobility Helm chart and their default values.
+
+| Parameter | Description | Default |
+| - | - | - |
+| `replicaCount` | Number of replicas for the Application Mobility controllers | `1` |
+| `image.pullPolicy` | Image pull policy for the Application Mobility controller images | `IfNotPresent` |
+| `controller.image` | Location of the csm-application-mobility Docker image | `dell/csm-application-mobility-controller:v0.1.0` |
+| `cert-manager.enabled` |  If set to true, cert-manager will be installed during Application Mobility installation | `false` |
+| `veleroNamespace` |  If Velero is already installed, set to the namespace where Velero is insatlled | `velero` |
+| `licenseName` |  Name of the Secret that contains the License for CSM Application Mobility | `license` |
+| `objectstore.secretName` |  If velero is already installed on the cluster, specify the name of the secret in velero namespace that has credentials to access object store | ` ` |
```

Comment:
Is this value optional? A column `Required <Yes|No>` would be helpful.

Reply:
Added Required column